joomla/joomla-cms
# [4.4] Untranslated string for warning message in update:extensions:check CLI

Pull Request for Issue #43206 .

### Summary of Changes
Language file is now loaded before outputting the warning message.

### Testing Instructions
Arrange for an extension update site to have an invalid URL (eg. temporarily mangle the URL in the extensions database). Then run `php cli/joomla.php update:extensions:check` from the command line.

### Actual result BEFORE applying this Pull Request
Warning message: `[WARNING] JLIB_UPDATER_ERROR_OPEN_UPDATE_SITE`

### Expected result AFTER applying this Pull Request
Warning message something like
`[WARNING] Update: Could not open update site #4 "xxxxxxxxxxx Package Update Site", URL: https://example.com/xxxxx/update.xml`

### Link to documentations
Please select:
